### PR TITLE
Added info about eZ Platform Demo

### PR DESCRIPTION
**Pull request by krzyraj**
Added description based on Slack conversation: https://ezcommunity.slack.com/archives/ezplatform-use/p1450429296001006

### CODE REVIEW

**Comment by clash82** [comment]
wow, thanks :)

File: README.md [hunk 1]
```diff
@@ -47,6 +47,9 @@ features for editorial teams and media companies, 100% built on top of *eZ Platf
 ## Install and Upgrade
 For installation & upgrade instructions, see [INSTALL.md](https://github.com/ezsystems/ezplatform/blob/master/INSTALL.md).
 
+### eZ Platform Demo
+This repository lets you create a clean, empty installation of eZ Platform. You can also choose a version of eZ Platform including a demo, that is an example website. It is available in the following repository: https://github.com/ezsystems/ezplatform-demo
```
Comment (bdunogier) [inline README.md]: I'd maybe add what this version is for (developing / building a project).

**Comment by krzyraj** [comment]
Updated description of eZ Platform Demo.

**Comment by andrerom** [comment]
+1

**Pull request merged**